src/DesignCompile/ResolveSymbols.cpp

```diff
@@ -65,13 +65,13 @@ void ResolveSymbols::createFastLookup() {
   // std::string fileName =  "FILE: " + m_fileData->getFileName() + " " +
   // m_fileData->getChunkFileName () + "\n"; std::cout << fileName;
 
-  std::vector<VObjectType> types = {
+  std::unordered_set<VObjectType> types = {
```

Comment:
These look like they can be 'static const', but that can probably  wait for another PR.